Added release notes for v5.9 (#134)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/release-notes.adoc

= Release notes

This release introduced bug fixes and improvements to existing features and UX.

> == New Features

Reviewer: ```suggestion
== New Features

```

- SQL configuration added to Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/831[#831])
- Added ability to set arbitrary JVM options in Management Center CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/801[#801])

== Changes
- SQL configuration added to Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/831[#831])

> - Added ability to set arbitrary JVM options in Management Center CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/801[#801])

Author: @Serdaro Thanks a lot for your comments. Can you please check 3 more new items under #Changes, I forgot to add them

> - Management Center config change from XML to YAML format (https://github.com/hazelcast/hazelcast-platform-operator/pull/823[#823])

Reviewer: ```suggestion
- Changed the Management Center configuration format from XML to YAML. (https://github.com/hazelcast/hazelcast-platform-operator/pull/823[#823])
```

> - Updated the status of a failed JetJob associated with a Hazelcast cluster that is not in a running state. (https://github.com/hazelcast/hazelcast-platform-operator/pull/799[#799])

Reviewer: ```suggestion
- Creating a JetJob for a Hazelcast cluster, which is not running, now fails. (https://github.com/hazelcast/hazelcast-platform-operator/pull/799[#799])
```

> - Added webhook validation for WanReplication updates and standardized error messages to ensure consistency with other validation (https://github.com/hazelcast/hazelcast-platform-operator/pull/803[#803])

Reviewer: ```suggestion
- Added webhook validation for WAN Replication updates, and standardized the error messages to ensure consistency with other validations. (https://github.com/hazelcast/hazelcast-platform-operator/pull/803[#803])
```
